unlock_indicator.c

```diff
@@ -291,10 +291,29 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
     }
 
     if (xr_screens > 0) {
```

Comment:
I realize you’re not changing this part of the code, but currently we have:

```c
if (xr_screens > 0) {
  for (int screen = 0; screen < xr_screens; screen++) {
    // …
  }
} else {
  // else
}
```

And to save one level of indentation (which will be good to make the code you’re adding more readable), we can pull out the for loop like so:

```c
for (int screen = 0; screen < xr_screens; screen++) {
  // …
}
if (xr_screens == 0) {
  // else
}
```